plugins/woocommerce/tests/legacy/unit-tests/coupon/data.php

```diff
@@ -174,7 +174,7 @@ public function test_coupon_getters_and_setters() {
 			'description'                => 'hello world',
 			'discount_type'              => 'percent',
 			'amount'                     => 10.50,
-			'status'                     => 'publish',
+			'status'					 => 'publish',
```

Comment:
@anitaa1990 I think the problem here is that there is a mix of tabs and spaces being used to align the `=>`. It should just be spaces.

```suggestion
		'status'                       => 'publish',
```